services/core-web/src/components/mine/DigitalPermitCredential/ViewDigitalPermitCredential.tsx

```diff
@@ -0,0 +1,344 @@
+import "@ant-design/compatible/assets/index.css";
+
+import { Alert, Button, Col, Row, Table, Typography } from "antd";
+import {
+  VC_ACTIVE_CONNECTION_STATES,
+  VC_CONNECTION_STATES,
+  VC_CRED_ISSUE_STATES,
+} from "@mds/common";
+import {
+  IMine,
+  IMineCommodityOption,
+  IMineDisturbanceOption,
+  IPermit,
+} from "@mds/common/interfaces";
+import React, { FC, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { formatDate } from "@mds/common/redux/utils/helpers";
+import {
+  renderDateColumn,
+  renderTextColumn,
+} from "@mds/common/components/common/CoreTableCommonColumns";
+import { useSelector, useDispatch } from "react-redux";
+import { getPermits } from "@mds/common/redux/selectors/permitSelectors";
+import { getMines } from "@mds/common/redux/selectors/mineSelectors";
+import ExplosivesPermitMap from "@mds/common/components/explosivespermits/ExplosivesPermitMap";
+import {
+  getMineCommodityOptions,
+  getMineDisturbanceOptions,
+} from "@mds/common/redux/selectors/staticContentSelectors";
+import { closeModal, openModal } from "@mds/common/redux/actions/modalActions";
+import modalConfig from "@/components/modalContent/config";
+import {
+  fetchCredentialConnections,
+  getCredentialConnections,
+  revokeCredential,
+} from "@mds/common/redux/slices/verifiableCredentialsSlice";
+
+const { Paragraph, Title } = Typography;
+
+const permitAmendmentLike = (permit: IPermit): any => ({
+  permit_amendment_id: undefined,
+  permit_amendment_guid: undefined,
+  amendment_no: 0,
+  ...permit,
+});
+
+export const ViewDigitalPermitCredential: FC = () => {
+  const dispatch = useDispatch();
+  const permits = useSelector((state) => getPermits(state));
+
+  const { permitGuid, id: mineGuid } = useParams<{
+    permitGuid: string;
+    id: string;
+  }>();
+
+  const connectionDetails = useSelector((state) => getCredentialConnections(state));
+  const digitalPermitCredential: IPermit = permits.find((p) => p.permit_guid === permitGuid);
+  const mines = useSelector((state) => getMines(state));
+  const mine: IMine = mines[mineGuid];
```

Comment:
if you want to, there is also getMineById and getPermitByGuid (I just love making selectors I guess. They do use the other ones under the hood).

Reply:
Nice, sounds good!